sagemaker-spark-sdk/src/main/scala/com/amazonaws/services/sagemaker/sparksdk/transformation/deserializers/ProtobufResponseRowDeserializer.scala

```diff
@@ -192,3 +192,16 @@ class FactorizationMachinesRegressorDeserializer
   extends ProtobufResponseRowDeserializer(schema =
     StructType(Array(StructField(scoreColumnName, DoubleType))),
     protobufKeys = Some(Seq("score")))
+
+/**
+  * Deserializes a Protobuf response from the LDA model image to a Vector of Doubles containing
```

Comment:
Suggestion:

"Vector of Doubles containing the projection of the input vector"
->
"Vector of Doubles representing the topic mixture for the document represented by the input vector"

Reply:
updating